docs/source/folder_structure.md

```diff
@@ -1,45 +1,68 @@
+    ├── rawdata/
+    │   └── sub-<label>/
+    │       └── ses-<label>/
+    │           ├── <datatype_1>/
+    │           │   └── raw_data.file
+    │           └── <datatype_2>/
+    │               └── raw_data.file  
+    └── derivatives/
+        └── ...
 ```
-## Naming folders
 
-### Project
-Can have any name, this should be descriptive of the dataset contained in the folder.
+The project folder may have any name descriptive of the project (that does not include spaces). Within the 
+project folder, data is separated into ```rawdata``` and ```derivatives```. ```rawdata``` contains the raw
+data acquired during data collection. ```derivatives``` includes any processed data (e.g. ephys preprocessing, 
+behavioural movement tracking). The subject / session / datatype folder strcture within derivatives should
+match the rawdata folder.
+
+Each subject (e.g. mouse, rat) in the project has a dedicated folder in which all data from 
+experimental sessions are stored.
+In the case of multiple sessions, each session has its own folder where acquired data, belonging to
+a dataype (e.g. ```ephys, behav, imaging, histology``) are stored.
 
-### Subject
-Structure: `sub-<participant label>`
-One folder per subject in this dataset. Labels should be unique for each subject.
-Example: `mouse-01`
+Subject and session folder names cannot contain spaces and must consist of key-value paires separated
+by underscores, e.g. ``sub-001_date-06122022``. 
 
-### Session
-Structure: `ses-<session label>`
+### subject
+The subject level is prefixed with the REQUIRED "sub" key and optional value, e.g. ```sub-001```. Only one subject
+folder per-subject is permitted, and subject labels must be unique to each subject.
 
-In general, a session represents a recording, imaging or behavioural experiment, with a definite kind of techniques involved and a given set of parameters. You might have multiple sessions per subject if you collected data from them on several occasions. 
-Example: `ses-01`
+### session
+If data for the subject were acquired across multiple sessions,then within the subject 
+directory resides subdirectories named with the REQUIRED prefix "ses", e.g. ```ses-<label>```. A session represents a recording, 
+imaging or behavioural experimental session, and may contain multiple blocks, or runs, of data acquisition.
+The session folder will contain the acquired datatypes. Different sessions may have different combinations of datatypes.
 
-### Datatype
-Structure: `datatype`
-Represents different types of data. Must be one of:
-- `ephys`
-- `imaging` (microscopy)
-- `behaviour`
-- `histology`
+### datatype
+Datatypes included at the session level are REQUIRED to have one of the following names ```ephys, behav, imaging, histology```.
+If collected, ```ephys, behav, imaging``` are REQUIRED to be placed at the session label. 
+If collected, ```histology``` is REQUIRED to be the subject level. 
+
+### Example
+A real project folder may look like: 
+```
+project
```

Comment:
Could you add the lines (not sure what they're called) to illustrate the directory structure?